src/Models/TaskVersion.php

```diff
@@ -95,4 +95,16 @@ public function getDeadlineAttribute($value): string
     {
         return Carbon::parse($value)->format('c');
     }
+
+    public function taskFiles(): HasMany
+    {
+        return $this->hasMany(TaskFile::class, 'task_id', 'task_id');
+    }
+
```

Comment:
This relationship is also problematic. Nowhere do we version comments when we version task. The Comment doesn't have a FK pointing toward the TaskVersion table.